cms/envs/common.py

```diff
@@ -2815,8 +2815,9 @@
 
 BRAZE_COURSE_ENROLLMENT_CANVAS_ID = ''
 
-DISCUSSIONS_INCONTEXT_FEEDBACK_URL = ''
-DISCUSSIONS_INCONTEXT_LEARNMORE_URL = ''
+# pylint: disable=line-too-long
+DISCUSSIONS_INCONTEXT_FEEDBACK_URL = "https://discuss.openedx.org/t/new-and-improved-discussions-forum/9183"
```

Comment:
These feedback URLs often lead to disappointing experiences for our users. It would be better to have them default to empty and then operators who are investing in a feedback loop can override as they like.

Reply:
Thanks for the review @pdpinch.
I made them default to empty, as per your suggestion.